tests/tracer/test_propagation.py

```diff
@@ -20,19 +20,242 @@
 from ddtrace.propagation.http import _HTTP_HEADER_B3_SINGLE
 from ddtrace.propagation.http import _HTTP_HEADER_B3_SPAN_ID
 from ddtrace.propagation.http import _HTTP_HEADER_B3_TRACE_ID
+from ddtrace.propagation.http import _HTTP_HEADER_TAGS
 
 from ..utils import override_global_config
 
 
 NOT_SET = object()
 
 
+def test_inject(tracer):
+    # We will only pass along `_dd.p.dm` tags
+    meta = {"_dd.p.dm": "value", "_dd.p.other": "value"}
```

Comment:
Need to verify that this is the expected behavior given that we still parse the tag that is not in the allowed list.

Reply:
We do not have an allow list for extract/inject.